### PR TITLE
Making the test cases easier for debugging. Changing function name to…

### PR DESCRIPTION
**Pull request by reolander**
… be obvious.

1. The question asks us to find sum of multiples of 3 or 5 but the function name is multiples 3 and 5. This may confuse the programming into
thinking that he needs to find sum of numbers which are multiples of both 3 and 5 (i.e. multiples of 15). 
2. Removed hidden test cases!

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by raisedadead — COMMENTED** [review]
Hi @reolander 

Thanks a lot for this PR.

Looks like a significant change. Needs discussion.

Has this changed been discussed before? For example, do you have references to any discussions on the forum or a related issue thread?

**Comment by reolander** [comment]
@raisedadead there was a discussion in my other PR about how test cases should be easier for debugging.

**Comment by raisedadead** [comment]
> there was a discussion in my other PR...

How about you link it in here? It makes our job much easier to follow the discussion. The more context we have the faster we can QA and merge things.

**Comment by reolander** [comment]
> 
> 
> > there was a discussion in my other PR...
> 
> How about you link it in here? It makes our job much easier to follow the discussion. The more context we have the faster we can QA and merge things.

https://github.com/freeCodeCamp/freeCodeCamp/pull/34722

**Review by lynxlynxlynx — APPROVED** [review]
LGTM

File: curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-1-multiples-of-3-and-5.english.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ Find the sum of all the multiples of 3 or 5 below the provided parameter value <
...
-    testString: assert.strictEqual(multiplesOf3and5(19564), 89301183, '<code>multiplesOf3and5(19564)</code> should return 89301183.');
-  - text: Your function is not returning the correct result using our tests values.
-    testString: assert.strictEqual(multiplesOf3and5(8456), 16687353, 'Your function is not returning the correct result using our tests values.');
+  - text: <code>multiplesOf3or5(1000)</code> should return 233168.
+    testString: assert.strictEqual(multiplesOf3or5(1000), 233168, '<code>multiplesOf3or5(1000)</code> should return 233168.');
+  - text: <code>multiplesOf3or5(49)</code> should return 543.
+    testString: assert.strictEqual(multiplesOf3or5(49), 543, '<code>multiplesOf3or5(49)</code> should return 543.');
+  - text: <code>multiplesOf3or5(19564)</code> should return 89301183.
+    testString: assert.strictEqual(multiplesOf3or5(19564), 89301183, '<code>multiplesOf3or5(19564)</code> should return 89301183.');
+  - text: Your function should return sum of all the multiples of 3 or 5 below the provided parameter value.
```
Comment (lynxlynxlynx) [inline curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-1-multiples-of-3-and-5.english.md]:
```suggestion
  - text: Your function should return the sum of all the multiples of 3 or 5 below the provided parameter value.
```

**Comment by RandellDawson** [comment]
Quick question here.  If we are going to change all the verbiage to `or` (including the front matter title), should we also change the filename to be `problem-1-multiples-of-3-or-5.english.md` ?

**Comment by lynxlynxlynx** [comment]
Doesn't seem that important to me.

**Comment by reolander** [comment]
@lynxlynxlynx Can you close this PR after having merged this with the master branch?

**Comment by lynxlynxlynx** [comment]
That happens automatically. Did you want something special?

**Review by ojeytonwilliams — COMMENTED** [review]
I think the [guide page](https://github.com/freeCodeCamp/freeCodeCamp/blob/6d43682397648f849c3bd41dc0d683be719b7829/guide/english/certifications/coding-interview-prep/project-euler/problem-1-multiples-of-3-and-5/index.md) would need to be updated.  Both because it uses `multiplesOf3And5` and because the challenge's hint url would no longer point to it.
  
As for this PR, LGTM :+1:

**Comment by scissorsneedfoodtoo** [comment]
@reolander, thank you for taking the time to submit this PR.

I'm of the opinion that we should leave the challenge title and function name the way they are since the actual Project Euler challenge is called "Multiples of 3 and 5". I can see how it can be a bit confusing, but the description does a good job explaining what the user should do. I suppose the title is referring to the answer being the sum of the multiples of 3 and 5.

Like @ojeytonwilliams mentioned, if we decide to make this change we would have to update the Guide page at the very least, and should then consider changing the filename for all the languages and in the [meta.json](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/_meta/project-euler/meta.json) file.

All that said, I like the new text you wrote for the test! Much better than the hidden test case before.

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo In an effort to make it clear for what you are expecting from @reolander, are you saying that only the `text` for the tests should be changed and nothing else?  If that is the case, then the `text` would still have the `or` in the `code` tags?  Before your last sentence, I was under the impression we should leave the original challenge as is(which I am for) and close this PR as not needed.

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo What are your thoughts on my last comment?

**Comment by scissorsneedfoodtoo** [comment]
@RandellDawson, sorry, I missed your earlier comment. Also could have been more clear.

I was just trying to say that, if enough contributors agree, then we should just change the `text` for the tests. But like you I would much prefer leaving the text as is. The only thing from this PR I would keep is the change from a hidden test case to an explicit one in the last test. That could be done in another PR, though.

**Pull request closed without merging**